src/controllers/mainController.ts

```diff
@@ -408,7 +411,7 @@ export default class MainController implements vscode.Disposable {
     /**
      * Verifies the extension is initilized and if not shows an error message
      */
-    private CanRunCommand(): boolean {
+    private canRunCommand(): boolean {
```

Comment:
>canRunCommand [](start = 12, length = 13)

nice, I just asked Kevin to rename this method in another code review.  I'll leave a comment for him to not do that to avoid conflicts